### ISSUE TITLE
Misidentified language

### ISSUE THREAD

**Issue by Techgirljen** [validator/validator#1779]
Hello, 
I am getting the message below; my code is in English.  

Warning: This document appears to be written in English. Consider adding lang="en" (or variant) to the html start tag.

From line 1, column 16; to line 2, column 6

type html>↩<html>↩<head

Here is my line of code:
<html lang="en">

How do I fix this? 
Thanks for your help!

**Comment by tenzap** [comment]
Can you provide a sample?

**Comment by j9t** [comment]
@Techgirljen, from the snippet you shared, this suggests to change

```html
<!doctype html>
<head>
<!-- ... -->
```

to

```html
<!doctype html>
<html lang="en">
  <head>
  <!-- ... -->
```

Does this help?

**Comment by j9t** [comment]
@Techgirljen, I’m feeling free to close this one. Please follow up if there’s still an issue, happy to review again. Cheers!
